docs/developer-guide/meta/dist-lock.md

```diff
@@ -0,0 +1,57 @@
+# Distributed Lock
+
+## Introduction
+
+Meta provides the ability to distribute lock externally via gRPC.
```

Comment:
```suggestion
`Meta` provides the functionality of distribute locks via gRPC.
```

Reply:
since provides is used right underneath, can we change this sentence to sth like "by using gRPC, 'Meta' is able to distribute lock "?